examples/cifar10/cifar10.py

```diff
@@ -108,7 +108,8 @@ def _variable_on_cpu(name, shape, initializer):
   """
   with tf.device('/cpu:0'):
     dtype = tf.float16 if FLAGS.use_fp16 else tf.float32
-    var = tf.get_variable(name, shape, initializer=initializer, dtype=dtype)
+    var = tf.get_variable(
+        name, shape, initializer=initializer, dtype=dtype)
```

Comment:
Do you know which rule is kicking in here?  It appears here and below, but seems overly aggressive and potentially worse in terms of readability...

Reply:
Interestingly, this does not happen when I run `autopep8` on this isolated fragment.

Reply:
Yeah, I couldn't figure out why autopep8 was doing this... except that ignoring E501 seemed to quiet it down.